utils/markdown.js

```diff
@@ -341,7 +340,9 @@ function innerRenderMdNode(indent, node, lastNode, result, options) {
   if (node.type === 'note') {
     newLine();
     result.push(`${indent}:::${node.noteType}`);
-    result.push(wrapText(node.text, options, indent));
+    const children = node.children || [];
+    for (const child of children)
+      innerRenderMdNode(indent, child, children[children.length - 1], result, options);
```

Comment:
Looking at other calls of `innerRenderMdNode`, I think that `lastNode` actually means `previousNode`, so you should pass the previous node from the `children` array.